examples/dtls-sock/Makefile

```diff
@@ -29,15 +29,15 @@ USEMODULE += shell_commands
 DTLS_PORT ?= 20220
 CFLAGS += -DDTLS_DEFAULT_PORT=$(DTLS_PORT)
 
-# NOTE: If no cipher suite is selected, DTLS_PSK is used by default.
+# NOTE: If no cipher suite is selected, CONFIG_DTLS_PSK is used by default.
 # This adds support for TLS_PSK_WITH_AES_128_CCM_8
-CFLAGS += -DDTLS_PSK
+CFLAGS += -DCONFIG_DTLS_PSK
```

Comment:
Maybe here add a comment saying that this should be commented out in case of using Kconfig.

Reply:
Added comment